lib/src/main/java/edu/stanford/futuredata/macrobase/ingest/CSVDataFrameLoader.java

```diff
@@ -57,23 +57,23 @@ public DataFrame load() throws Exception {
         this.badRecords = 0;
         ArrayList<Row> rows = new ArrayList<>();
         for (CSVRecord record : csvParser) {
-            try {
-                ArrayList<Object> rowFields = new ArrayList<>(numColumns);
-                for (int c = 0; c < numColumns; c++) {
-                    Schema.ColType t = columnTypeList[c];
-                    String rowValue = record.get(c);
-                    if (t == Schema.ColType.STRING) {
-                        rowFields.add(rowValue);
-                    } else if (t == Schema.ColType.DOUBLE) {
+            ArrayList<Object> rowFields = new ArrayList<>(numColumns);
+            for (int c = 0; c < numColumns; c++) {
+                Schema.ColType t = columnTypeList[c];
+                String rowValue = record.get(c);
+                if (t == Schema.ColType.STRING) {
+                    rowFields.add(rowValue);
+                } else if (t == Schema.ColType.DOUBLE) {
+                    try {
                         rowFields.add(Double.parseDouble(rowValue));
-                    } else {
-                        throw new RuntimeException("Bad ColType");
+                    } catch (NumberFormatException e) {
+                        rowFields.add(Double.NaN);
```

Comment:
Yeah, when a certain value in a double column was NaN (like standard deviation), the row would be thrown out